www/templates/html/SiteMaster/Core/Auditor/Site/Review/EditForm.tpl.php

```diff
@@ -0,0 +1,77 @@
+<?php
+use \SiteMaster\Core\Auditor\Site\Review;
+
+/**
+ * @var $context \SiteMaster\Core\Auditor\Site\Review\EditForm
+ */
+?>
+
+<form method="POST">
+    <ul>
+        <li>
+            <?php 
+            $date_scheduled = '';
+            if ($context->review) {
+                $date_scheduled = date('Y-m-d', strtotime($context->review->date_scheduled));
```

Comment:
I see this pattern throughout the project, but think it would be a good idea to build something into the model that will parse and store the database datetime a single time. This will avoid unnecessary overhead of parse the same time string over and over. Moreover, I'd like to see that internal storage be of type `\DateTime` which will hopefully be more future proof than the procedural style functions.
Example: `$context->review->getDateScheduled()->format('Y-m-d')`

Obviously this is not an immediate need, but something I think that would benefit performance and readability.